### ISSUE TITLE
Deployment fails when not supplying a runbook or module to import

### ISSUE THREAD

**Issue by tw3lveparsecs** [tw3lveparsecs/azure-bicep-automation-account#1]
cannot deploy automation account with no modules or runbooks.
``` bicep
param deploymentName string = concat('automationAccount', utcNow())

module automationAccount '../azure-bicep-automation-account/main.bicep' = {
  name: deploymentName  
  params: {
    name: 'MyAutomationAccount'
  }
}
```

**Comment by tw3lveparsecs** [comment]
this is a current limitation of Bicep and will be resolved with bicep v0.4 https://github.com/Azure/bicep/issues/1754

**Comment by tw3lveparsecs** [comment]
solved with Azure/bicep#1754
